pytrickle/stream_processor.py

```diff
@@ -41,6 +47,10 @@ def from_handlers(
             attr = getattr(handler_instance, attr_name)
             info = getattr(attr, "_trickle_handler_info", None)
             if not info:
+                func_obj = getattr(attr, "__func__", attr)
+                func_obj = inspect.unwrap(func_obj)
+                info = getattr(func_obj, "_trickle_handler_info", None)
+            if not info:    
```

Comment:
Remove trailing whitespace
```suggestion
            if not info:
```